### DISCUSSION TITLE
About Japanese Language Support

### DISCUSSION THREAD

**Discussion by kunimaeda** [bacula-web/bacula-web#201] [Q&A] [answered]
Dear team, Nice to meet you, I am a user from Japan. My name is Kuni.
I recently installed bacula and baculum on Rocky Linux 9.3 with Docker.
After installation, we finished setting up the API, set up the web, and set the language to Japanese.
However, Japanese is not displayed correctly. All displays look like @@Dashboard@@. Please let me know how to solve this problem.
Kuni.
Best regards.

**Comment by dfranco** [comment, marked as answer]
Hello @kunimaeda ,

If you need help for baculum, Its better if you write to bacula-users mailing list.

This repo is dedicated to bacula-web project.

Best,

**Reply by kunimaeda** [reply]
Hello Mr. Davide Franco
Thank you for your advice.
I will ask it to the mailing list.
Best regards.

**Reply by dfranco** [reply]
> Hello Mr. Davide Franco
> 
> Thank you for your advice.
> 
> I will ask it to the mailing list.
> 
> Best regards.

You're welcome
